SpatialGDK/Documentation/content/get-started/dependencies.md

```diff
@@ -0,0 +1,27 @@
+# Get started: 1 - Get the Dependencies
+
+### Hardware
+* Refer to the <a href="https://docs.unrealengine.com/en-US/GettingStarted/RecommendedSpecifications" data-track-link="Clicked UE4 Recommendations|product=Docs|platform=Win" target="_blank">UE4 recommendation</a> (Unreal documentation)
+* Recommended storage: 60GB+ available space
+
+### Network settings
+* Refer to the [SpatialOS network settings](https://docs.improbable.io/reference/latest/shared/get-started/requirements#network-settings) (SpatialOS documentation)
```

Comment:
WE're putting the " (SpatialOS documentation)" into the link now so:

[SpatialOS network settings (SpatialOS documentation)]

Reply:
Full stop?

Reply:
done